dashboard/app/controllers/openai_chat_controller.rb

```diff
@@ -72,14 +72,27 @@ def add_content_to_system_prompt(system_prompt, level_instructions, test_file_co
   end
 
   private def read_file_from_s3(key_path)
-    if [:development, :test].include?(rack_env)
+    cache_key = "s3_file_#{key_path}"
```

Comment:
@ebeastlake I don't know much about caching besides "we should probably do it here". It makes sense to me to run it on prod and maybe test too? I don't see a downside to running caching on test.